warehouse/static/sass/blocks/_project-description.scss

```diff
@@ -55,16 +55,6 @@
 
   // Remove top margin on elements at the top of the description
   > div:first-child > *:first-child,
-  > *:first-child,
```

Comment:
I do not understand this... do we know why it was repeated before?

Reply:
Originated during this change: https://github.com/pypi/warehouse/commit/a5be8a7682e735a7e7391d582461c470a06de711